executor/server.go

```diff
@@ -131,6 +144,15 @@ func (s *Server) Stop() {
 		}
 	}
 
+	if s.metastore != nil {
```

Comment:
Can you add a comment why it is necessary to call `Delete` here?

Reply:
added comment